app/controllers/catalog_controller.rb

```diff
@@ -45,6 +45,21 @@ def index
     end
   end
 
+  def track
```

Comment:
overriding this method to add document_id 👍   I know you haven't tidy up the PR yet but just thought i'd mention it would be nice to add a comment to point that it is an override.